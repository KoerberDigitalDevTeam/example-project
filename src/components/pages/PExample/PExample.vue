<template>
  <TMainPage v-testId.example-page :title="$t('examplePage.title')">
    <div v-testId.example-page.content class="p-example__content">
      {{ $t('examplePage.content') }}
    </div>
    <template #footer v-testId.example-page.footer>{{ $t('examplePage.footer') }}</template>
  </TMainPage>
</template>

<script>
import TMainPage from '@/components/templates/TMainPage/TMainPage.vue';

export default {
  name: 'PExample',

  components: {
    TMainPage,
  },
};
</script>

<style lang="scss" scoped>
.p-example__content {
  display: flex;
  height: 100%;
}
</style>
