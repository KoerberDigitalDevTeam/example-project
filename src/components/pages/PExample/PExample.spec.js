import { shallowMount } from '@vue/test-utils';
import Vuetify from 'vuetify';
import PExample from './PExample.vue';

function createWrapper(options = {}) {
  return shallowMount(PExample, {
    vuetify: new Vuetify(),
    ...options,
  });
}

describe('PExample', () => {
  it('Should render PExample', () => {
    const wrapper = createWrapper();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
