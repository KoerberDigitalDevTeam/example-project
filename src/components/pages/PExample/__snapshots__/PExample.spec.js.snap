// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PExample Should render PExample 1`] = `
<tmainpage-stub title="examplePage.title" data-testid="example-page">
  <div class="p-example__content" data-testid="example-page-content">
    examplePage.content
  </div>
</tmainpage-stub>
`;
